CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+# Change log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com) and this project adheres to [Semantic Versioning](http://semver.org).
+
+## 0.3.6 - 2017-03-04
+### Added
+- Add CHANGELOG.md.
+
+## 0.3.5 - 2017-03-02
+### Added
+- Add pull request template document. 
+
+## 0.3.4 - 2017-03-01
+### Added
+- Add test case to test webpack plugin workflow by checking build result.
+
+## 0.3.3 - 2017-02-21
+### Changed
+- Update README.md.
+
+## 0.3.2 - 2017-02-21
+### Added
+- Add unit test case using webpack mock.
+
+## 0.3.1 - 2017-02-21
+### Changed
+- Update package dependencies for eslint and eslint plugin.
+
+## 0.3.0 - 2017-01-31
+### Added
+- Add unit test with code coverage.
+
+### Changed
+- Update contributing project status in README.md.
+
+## 0.2.4 - 2017-01-25
+### Added
+- Add silent and buildTrigger options.
+
+### Changed
+- Update eslint package version.
+
+## 0.2.3 - 2017-01-16
+### Changed
+- Update package dependencies for eslint and eslint plugin.
+
+## 0.2.2 - 2016-12-19
+### Changed
+- Update eslint package version.
+- Fix bugs, checking the files exist before reading files.
+
+## 0.2.1 - 2016-12-16
+### Changed
+- Divide the logic of manual call to replace content.
```

Comment:
This modification was made for this reason:
the plugin can be called outside a webpack workflow, that's why I extracted replace method to not call apply method.

Could you replace this change entry by:
`Extract replace method to be called outside a webpack workflow` ?